stable/jaeger-operator/templates/role.yaml

```diff
@@ -0,0 +1,63 @@
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: {{ include "jaeger-operator.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "jaeger-operator.name" . }}
+    helm.sh/chart: {{ include "jaeger-operator.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+rules:
+- apiGroups:
+  - io.jaegertracing
+  resources:
+  - "*"
+  verbs:
+  - "*"
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - services
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  - configmaps
+  - secrets
+  verbs:
+  - "*"
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  - daemonsets
+  - replicasets
+  - statefulsets
+  verbs:
+  - "*"
+- apiGroups:
+  - extensions
+  resources:
+  - ingresses
+  verbs:
+  - "*"
+
+---
+{{- if .Values.rbac.create }}
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1beta1
+metadata:
+  name: default-account-{{ include "jaeger-operator.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "jaeger-operator.name" . }}
+    helm.sh/chart: {{ include "jaeger-operator.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+subjects:
+- kind: ServiceAccount
+  name: default
```

Comment:
See https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md for how to define the serviceaccount.

Reply:
thanks for the review and the link. Done. PTAL @davidkarlsen